packages/overlay/README.md

```diff
@@ -345,19 +231,117 @@ When nesting multiple overlays, it is important to ensure that the nested overla
 </div>
 ```
 
-## Styling
+#### Focus Management
+
+The overlay manages focus based on its type:
+
+-   For `modal` and `page` types, focus is always trapped within the overlay
+-   For `auto` and `manual` types, focus behavior is controlled by the `receives-focus` attribute
+-   For `hint` type, focus remains on the trigger element
+
+Example of proper focus management:
+
+```html
+<sp-button id="modal-trigger" aria-haspopup="dialog" aria-expanded="false">
+    Open Settings
+</sp-button>
+<sp-overlay trigger="modal-trigger@click" type="modal">
+    <sp-dialog-wrapper
+        headline="Settings"
+        dismissable
+        underlay
+        aria-labelledby="settings-heading"
+    >
+        <h2 id="settings-heading" slot="heading">Settings</h2>
+        <sp-field-label for="setting1">Email Notifications</sp-field-label>
+        <sp-switch id="setting1">Enable notifications</sp-switch>
+
+        <div slot="footer">
+            <sp-button
+                variant="secondary"
+                onclick="this.dispatchEvent(new Event('close', { bubbles: true }))"
+            >
+                Cancel
+            </sp-button>
+            <sp-button
+                variant="accent"
+                onclick="this.dispatchEvent(new Event('close', { bubbles: true }))"
+            >
+                Save
+            </sp-button>
+        </div>
+    </sp-dialog-wrapper>
+</sp-overlay>
+```
+
+#### Keyboard Navigation
+
+-   ESC key closes overlays in reverse order of opening
+-   TAB/Shift+TAB navigates through focusable elements within modal/page overlays
+-   Arrow keys navigate through menu items in menu overlays
+-   ENTER/SPACE activates buttons and controls
+
+#### Screen Reader Considerations
+
+-   Use `aria-haspopup` on trigger elements to indicate the type of overlay
+-   Provide descriptive labels using `aria-label` or `aria-labelledby`
+-   Use proper heading structure within overlays
+-   Ensure error messages are announced using `aria-live`
+
+Example of a tooltip with proper screen reader support:
+
+```html
+<sp-button id="help-trigger" aria-describedby="help-tooltip" label="Help">
+    <sp-icon-help slot="icon"></sp-icon-help>
+</sp-button>
+<sp-overlay trigger="help-trigger@hover" type="hint">
+    <sp-tooltip id="help-tooltip">
+        Click for more information about this feature
+    </sp-tooltip>
+</sp-overlay>
+```
+
+### Advanced Topics
```

Comment:
Can we move advanced topics above Accessibility? That way consumers always know to look for Accessibility at the bottom.